scripts/db_dump_load.sh

```diff
@@ -21,6 +21,6 @@ bundle exec rails db:environment:set
 
 # Si vous avez besoin de débugger et que l'anonymisation complète vous bloque,
 # vous devez au moins anonymiser les données usager avec :
-# bundle exec rails runner 'Anonymizer::Core.anonymize_user_data!'
+# bundle exec rails runner 'Anonymizer.anonymize_user_data!'
 
 bundle exec rails runner scripts/anonymize_database.rb
```

Comment:
Il faut désormais passer le nom du schema à ce que je constate :

```suggestion
bundle exec rails runner scripts/anonymize_database.rb public
```